### ISSUE TITLE
Make app keyboard navigable

### ISSUE THREAD

**Issue by brandonhaslegs** [radicle-dev/radicle-interface#300]
Maybe someone has an idea of what keyboard shortcuts would be helpful and could advise here? And should add a Keyboard Shortcuts modal that shows all available keyboard shortcuts.

And just standard keyboard navigation should be implemented. Tabbing through all UI elements, ESC closing all modals, ESC unfocusing fields, being able to select anything you can select with a mouse.

**Comment by rudolfs** [comment]
Some hotkeys that come to mind:
  - <kbd>esc</kbd> to close a modal
  - <kbd>enter</kbd> do the primary action in a modal (e.g. click the next button)
  - <kbd>?</kbd> show what keyboard hotkeys are available
  - <kbd>/</kbd> focus the search bar
  - <kbd>t</kbd> fuzzy search for files in a project (we'd need to implement the feature first)

**Comment by brandonhaslegs** [comment]
Here’s an idea how it could look. Should match the standard modal design. File is here: https://www.figma.com/file/IlFjPJ5DjFa4EdvVmLVeKQ/%F0%9F%8C%B1-Radicle-RELOADED?node-id=13951%3A230801&t=uPjhJJDgSfR8auQG-4

<img width="879" alt="image" src="https://user-images.githubusercontent.com/4406983/208773429-2e2548f1-3b98-4577-a966-293a71d4efd0.png">

**Comment by cloudhead** [comment]
Should we maybe make the keys look like the search bar rather?

<img width="232" alt="image" src="https://user-images.githubusercontent.com/40774/209813922-2293a081-0b26-4722-9f4f-c159ebc540a4.png">

**Comment by cloudhead** [comment]
The way github renders them is also nice: 
<img width="39" alt="image" src="https://user-images.githubusercontent.com/40774/209814010-7033952b-a7f1-4a87-91e7-11952a283750.png">

**Comment by brandonhaslegs** [comment]
True they should match for sure! I also like the 3d style

**Comment by brandonhaslegs** [comment]
Made em 3d. [Link](https://www.figma.com/file/IlFjPJ5DjFa4EdvVmLVeKQ/%F0%9F%8C%B1-Radicle-RELOADED?node-id=13951%3A230801&t=QWCHS3N6eyq4uOYQ-1)

And made a few layout options
![image](https://user-images.githubusercontent.com/4406983/209993107-d0405c7c-0e0d-45ad-85fb-9e54e1a89a31.png)
![image](https://user-images.githubusercontent.com/4406983/209993118-227e74f0-795d-4da5-91b5-2c8d5082d319.png)
![image](https://user-images.githubusercontent.com/4406983/209993180-b6f659a9-887f-4d58-a6be-3aa14e2d14e0.png)
![image](https://user-images.githubusercontent.com/4406983/209993353-30629444-5de7-425c-84e9-73fa9253d165.png)

**Comment by cloudhead** [comment]
I like the first layout, but without the table borders :)

**Comment by brandonhaslegs** [comment]
Without the bounding box it feels weirdly aligned, but here it is. 
![image](https://user-images.githubusercontent.com/4406983/210558293-46dffcd4-f2f3-4a10-9063-5b504518c557.png)

**Comment by rudolfs** [comment]
Could you also provide a mockup on how the search bar hotkey should look like?

<img width="378" alt="Screenshot 2023-01-05 at 12 00 52" src="https://user-images.githubusercontent.com/158411/210764976-73d87a6a-969b-4c71-9c54-627f6c3cc1be.png">

**Comment by brandonhaslegs** [comment]
Here ya go. 
<img width="445" alt="image" src="https://user-images.githubusercontent.com/4406983/210783972-723fa176-09c8-410c-bd22-8f42dd8146f6.png”>

Here you can check out the details in Figma” https://www.figma.com/file/IlFjPJ5DjFa4EdvVmLVeKQ/%F0%9F%8C%B1-Radicle-RELOADED?node-id=14008%3A52230&t=WSRqOZ5Ox74kigqP-4

i hope we can change the logo to be the standard token logo one day 🥺

**Comment by cloudhead** [comment]
> i hope we can change the logo to be the standard logo one day pleading_face cc @cloudhead

idk, it looks too much like a token logo to me, not an app logo :/

**Comment by cloudhead** [comment]
> Without the bounding box it feels weirdly aligned, but here it is.

Yeah, perhaps having the text next to the keyboard buttons aligned would help.
